public/app/features/plugins/admin/components/PluginList.tsx

```diff
@@ -17,28 +15,15 @@ interface Props {
 
 export const PluginList = ({ plugins, displayMode }: Props) => {
   const isList = displayMode === PluginListDisplayMode.List;
-  const styles = useStyles2(getStyles);
   const { pathname } = useLocation();
   const pathName = config.appSubUrl + (pathname.endsWith('/') ? pathname.slice(0, -1) : pathname);
-
   return (
-    <div className={cx(styles.container, { [styles.list]: isList })} data-testid="plugin-list">
-      {plugins.map((plugin) => (
-        <PluginListItem key={plugin.id} plugin={plugin} pathName={pathName} displayMode={displayMode} />
-      ))}
+    <div data-testid="plugin-list">
+      <Grid display="grid" gap={3} templateColumns={isList ? '1fr' : 'repeat(auto-fill, minmax(288px, 1fr))'}>
```

Comment:
Should the Grid be able to accept this attributes instead (also `aria-label`, among others), so we don't need to introduce these kinds of wrappers?

Reply:
@JoaoSilvaGrafana and I talked about this and agreed on not accepting them at this moment. We'll let devs use this component and give feedback, including if they need these attributes in the Grid. Correct me if I'm wrong @JoaoSilvaGrafana.